stack/packaging-third-party-eups-dependencies.rst

```diff
@@ -22,10 +30,40 @@ See `this page`_ for a list of compatible licenses.
 .. _this page: https://www.gnu.org/licenses/license-list.html
 
 
+.. _fork-creating:
+
+Creating A Forked Repo
+======================
+
+A forked repo relies on building the package from source, rather than using binary artifacts, on the assumption that we may need to use the latest developments.
+Accordingly, we fork the package on GitHub into the ``lsst`` organization and add the "DM Externals" and "Overlords" teams to it as described in :doc:`adding-a-new-package`.
+This ensures that it will be tagged appropriately for official releases (while not conflicting with the third-party tags).
+
+Since the third-party package likely has no support for eups, we need to add a ``ups`` directory.
+This directory should contain the package's EUPS table file as well as an optional file :file:`eupspkg.cfg.sh` which will contain any customized commands for building and installing the third-party package.
+
+The upstream source has no need to know about the ``ups`` directory, so we put it on a separate branch in the ``lsst`` fork.
```

Comment:
I realize that I'm largely alone in this, but if we have some control over upstream, we can add ups support there and just track main, no?

Either way, if a maintainer has access to both, I'm wondering if it's better to put PRs on the fork or upstream first, given the choice.

Reply:
If you have access to both, it depends on the upstream merge latency (or willingness to have the Science Pipelines `lsst-dev` branch being based on a branch).